src/main/java/reposense/util/FileUtil.java

```diff
@@ -121,8 +151,27 @@ public static void copyDirectoryFiles(Path src, Path dest) throws IOException {
         }
     }
 
+    /**
+     * Zips all the Json files present in the {@code sourcePath}.
+     */
+    public static void zipJson(Path sourcePath) {
+        FileUtil.zip(sourcePath, ".json");
+    }
+
+    /**
+     * Returns a list of {@code Path} of {@code fileType} contained in the given {@code directoryPath} directory.
+     */
+    private static List<Path> getFilePaths(Path directoryPath, String... fileType) throws IOException {
+        List<Path> filteredFilePaths = new ArrayList<>();
+        for (String extension : fileType) {
+            filteredFilePaths.addAll(Files.walk(directoryPath)
+                    .filter(p -> p.toString().endsWith(extension))
```

Comment:
Write a helper method for the filtering, so that we can collect all inside the list at once, instead of having to go through each `extension`.